package.json

```diff
@@ -3,16 +3,17 @@
   "version": "1.0.0",
   "description": "Tell Slack you're working from home today 🏡",
   "repository": {
-	  "type": "git",
```

Comment:
funny you ask... the file i cloned had tabs and spaces mixed in... i changed the tabs that were out of place to regular spaces.  let me double check to make sure i didnt mess it up.